src/analytics.js

```diff
@@ -0,0 +1,100 @@
+define([], function () {
+    function _workflow(message) {
+        if (window.analytics) {
+            window.analytics.workflow(message);
+        }
+    }
+
+    function _usage(label, group, message) {
+        if (window.analytics) {
+            window.analytics.usage(label, group, message);
+        }
+    }
+
+    function _fbUsage(group, message) {
+        _usage("Form Builder", group, message);
+    }
+
+    function clickedSave() {
+        _workflow("Clicked Save in the form builder");
+    }
+
+    function clickedGoToQuestion() {
```

Comment:
Thanks. I like that much more. Got a little too ahead of myself trying to extract them all into functions. There were only a couple that were duplicated across files